dotcom-rendering/src/components/SignInGateSelector.importable.tsx

```diff
@@ -300,6 +304,17 @@ const SignInGateSelectorAuxia = ({
 			document.dispatchEvent(
 				new CustomEvent('article:sign-in-gate-dismissed'),
 			);
+
+			if (signInGateVersion === 'v2') {
+				// Emit modal dismiss event
+				document.dispatchEvent(
+					new CustomEvent('modal-dismissed', {
```

Comment:
LGTM, my only minor comment would be to possibly namespace the event e.g. `modal:close` and subsequently `modal:open` so that as a general rule of thumb, there's a differentiation between event types and event naming

Reply:
Good point, I'll make that change